tests/ProtoTests/BasicLro/Tests/BasicLroClientTest.php

```diff
@@ -118,5 +121,36 @@ public function method1ExceptionTest()
         $incompleteOperation->setName('operations/method1ExceptionTest');
         $incompleteOperation->setDone(false);
         $transport->addResponse($incompleteOperation);
+        $status = new stdClass();
+        $status->code = Code::DATA_LOSS;
+        $status->details = 'internal error';
+        $expectedExceptionMessage = json_encode([
+            'message' => 'internal error',
+            'code' => Code::DATA_LOSS,
+            'status' => 'DATA_LOSS',
+            'details' => [],
+        ], JSON_PRETTY_PRINT);
+        $operationsTransport->addResponse(null, $status);
+        // Mock request
+        $response = $client->method1();
+        $this->assertFalse($response->isDone());
+        $this->assertNull($response->getResult());
+        $expectedOperationsRequestObject = new GetOperationRequest();
+        $expectedOperationsRequestObject->setName('operations/method1ExceptionTest');
+        try {
+            $response->pollUntilComplete([
+                'initialPollDelayMillis' => 1,
+            ]);
+            // If the pollUntilComplete() method call did not throw, fail the test
+            $this->fail('Expected an ApiException, but no exception was thrown.');
+        } catch (ApiException $ex) {
+            $this->assertEquals($status->code, $ex->getCode());
+            $this->assertEquals($expectedExceptionMessage, $ex->getMessage());
+        }
+// Call popReceivedCalls to ensure the stubs are exhausted
```

Comment:
Nit: Align with lines below?

Reply:
Yes, this formatting is wrong ;)
This is generated code, and the problem is due to the code formatter being used, and there's a TODO to fix it. For the moment I'll leave it as it is, but it'll definitely get fixed soonish.